libs/journeys/ui/src/components/TextResponse/TextResponse.tsx

```diff
@@ -162,6 +163,7 @@ export const TextResponse = ({
                     'aria-labelledby': 'textResponse-label',
                     maxLength: 1000,
                     readOnly: selectedBlock !== undefined,
+                    inputMode: type === 'phone' ? 'tel' : 'text',
```

Comment:
```suggestion
                    inputMode: type === TextResponseType.phone ? 'tel' : 'text',
```
Can you use the reference to the enum instead of the string?